### PR TITLE
[Components with more than 1 service] LPS-188721 Refactor PortletPreferencesModelListener component

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-188721

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
Last push fixes sf problems

**Comment by MrJohn1911** [comment]
ci:test:stable

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

**Review by rafaprax — APPROVED** [review]

File: modules/apps/portal-configuration/portal-configuration-module-configuration/src/main/java/com/liferay/portal/configuration/module/configuration/internal/PortletPreferencesConfigurationListener.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.configuration.module.configuration.internal;
+
```
Comment (rafaprax) [inline modules/apps/portal-configuration/portal-configuration-module-configuration/src/main/java/com/liferay/portal/configuration/module/configuration/internal/PortletPreferencesConfigurationListener.java:7]: Hey @MrJohn1911, move this class to the proper package. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

**Review by rafaprax — APPROVED** [review]

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
ci:forward:force

cc @tinatian

**Pull request closed without merging**